src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -249,6 +252,17 @@ def _get_variance(self, timestep, prev_timestep):
 
         return variance
 
+    def _batch_get_variance(self, t, prev_t):
```

Comment:
Do we need these modifications when we have them already in the parallel versions?